README.markdown

```diff
@@ -13,6 +13,7 @@ non-blocking event-driven model (similar to the
 Implemented WebSocket protocol versions are:
 
  * [RFC 6455](http://tools.ietf.org/html/rfc6455)
+ * [RFC 7692](http://tools.ietf.org/html/rfc7692)
```

Comment:
Since we do not have PerMessageDeflate enabled by default, we should maybe point to a example in the wiki here